src/plugin.js

```diff
@@ -1,3 +1,4 @@
+/* global DocumentFragment */
```

Comment:
I'm making other changes based on feedback anyways, I'll bring in global and fix it properly.